core/src/main/groovy/com/novoda/gradle/release/ReleasePlugin.groovy

```diff
@@ -19,6 +20,28 @@ class ReleasePlugin implements Plugin<Project> {
 
     void attachArtifacts(PublishExtension extension, Project project) {
         if (project.plugins.hasPlugin('com.android.library')) {
+            def current = GradleVersion.current()
+            def version3_3 = GradleVersion.version("3.3")
+
+            if ((current <=> version3_3) <= 0) {
+                // detected Gradle 3.3 or smaller
+                // show message for required gradle upgrade
+                // details: https://github.com/novoda/bintray-release/issues/112
+                System.err.println("""
```

Comment:
The problem with failing in configuration phase is that you can't even upgrade with `./gradlew wrapper --gradle-version 3.5` because it will fail, too.